perftestnotebook.py

```diff
@@ -206,5 +212,40 @@ def main():
         json.dump(ptnb.fmt_data, f, indent=4, sort_keys=True)
 
 
-if __name__ == "__main__":
+    # Upload template file to Iodide
+    template = 'testing/template/template.txt'
+    tdata = ''
+    with open(template, 'r') as f:
+        tdata = f.read()
+
+    html = ''
+    with open('template_upload_file.html', 'r') as f:
+        html = f.read()
+
+    html = html.replace('replace_me', repr(tdata))
+    with open('upload_file.html', 'w+') as f:
+        f.write(html)
+    
+    webbrowser.open_new_tab('upload_file.html')
```

Comment:
Place this and everything below (down to app.run) into an `if 'analysis` in config:` condition so it only runs if there was an analysis was specified.